src/main/java/frc/robot/Robot.java

```diff
@@ -79,7 +79,14 @@ public void teleopInit() {
   /** This function is called periodically during operator control. */
   @Override
   public void teleopPeriodic() {
-
+    if(RobotContainer.xboxController.getAButton()){
+      PhotonVision.camera.setPipelineIndex(VisionConstants.kAprilPipline);
+      System.out.println("april tag pipeline");
+    }
+    else if(RobotContainer.xboxController.getBButton()){
```

Comment:
isn't it better to just write two separate "if" statements?

Reply:
they're the same, not critical